test/src/tests-forwarders.js

```diff
@@ -120,11 +122,10 @@ describe('forwarders', function() {
         window.mParticle.identifyRequest = {};
 
         done();
+        });
     });
 
     it('should permit forwarder if no consent configured.', function(done) {
-        mParticle._resetForTests(MPConfig);
```

Comment:
Why are we removing this?

Reply:
Since the init is removed, it is safe to remove the `resetForTests`.  If I didn't remove from others, it's fine to keep.  Just extraneous.  I'll add it as a cleanup ticket (though it could be its own as well)